SimplaeWAWS.Models/ApiApp.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Newtonsoft.Json;
```

Comment:
I don't really understand what you mean. Ideally they would stay in where they were if possible.